scripts/jestSetup.js

```diff
@@ -1,3 +1,6 @@
+require('core-js/stable');
+require('regenerator-runtime/runtime');
```

Comment:
What's this for? I would think jest would handle this for us?

Reply:
Whoops, you're right. I only need `core-js/stable` for `Object.entries`. I don't quite remember why I added this 😬

Reply:
May be good to add just the one polyfill directly so we don't add more than we need (and cause tests to run slower).

Can check out mms' polyfill.js file to see how we do it.